Mono.Cecil.csproj

```diff
@@ -253,6 +253,7 @@
   <ItemGroup>
     <None Include="mono.snk" />
   </ItemGroup>
+  <Import Project="Mono.Cecil.Settings" />
```

Comment:
That's a good point, but it does work in the Mac. I'll amend my latest commit to address this.